onionshare/__init__.py

```diff
@@ -51,6 +51,7 @@ def main(cwd=None):
     parser.add_argument('--connect-timeout', metavar='<int>', dest='connect_timeout', default=120, help="Give up connecting to Tor after a given amount of seconds (default: 120)")
     parser.add_argument('--stealth', action='store_true', dest='stealth', help="Use client authorization (advanced)")
     parser.add_argument('--receive', action='store_true', dest='receive', help="Receive shares instead of sending them")
+    parser.add_argument('--website', action='store_true', dest='website', help=strings._("help_website"))
```

Comment:
As of OnionShare 2.1, we no longer are using translatable strings in the CLI version (see the above strings which are now just hardcoded in English).

Are you able to replace your translatable strings to be hardcoded English strings like the above?

And in fact, `help_website` is already missing from the locale `en.json` so it doesn't work via CLI right now :)